libpod/oci.go

```diff
@@ -151,8 +151,6 @@ type OCIRuntime interface {
 type ExecOptions struct {
 	// Cmd is the command to execute.
 	Cmd []string
-	// CapAdd is a set of capabilities to add to the executed command.
-	CapAdd []string
```

Comment:
I think you need to retain this - `podman exec --privileged` can be run in containers that are not privileged.